docs-content/installing.html.md.erb

```diff
@@ -73,34 +71,74 @@ $ cf curl /v2/info
+        $ cf target -o appdynamics-org -s appdynamics-space
+        $ cf apps
+        (filtered)
+        appdnozzle-<version>
+        appdpcfdashboard-<version>
+
+1. Navigate to the AppDynamics Nozzle app and set the specific version of the tile deployed:  
  
-    $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_HOST host
-    $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_PORT port
-    $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_USERNAME user
-    $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_PASSWORD password
+        $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_HOST host
+        $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_PORT port
+        $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_USERNAME user
+        $ cf set-env appdnozzle-<version> APPD_SDK_CONTROLLER_HTTP_PROXY_PASSWORD password
  
- AppDynamics KPI Dashboard app:
+1. Navigate to the AppDynamics KPI Dashboard app and set the specific version of the tile deployed:
  
-    $ cf set-env appdpcfdashboard-<version> HTTP_PROXY http://user@password:host:port/
-    $ cf set-env appdpcfdashboard-<version> HTTPS_PROXY https://user@password:host:https_port/
+        $ cf set-env appdpcfdashboard-<version> HTTP_PROXY http://user@password:host:port/
+        $ cf set-env appdpcfdashboard-<version> HTTPS_PROXY https://user@password:host:https_port/
  
- Restage the apps for the environment variables to take effect:
+1. Restage the apps for the environment variables to take effect:
+
+        $ cf restage appdnozzle-<version>
+        $ cf restage  appdpcfdashboard-<version>
+
+### Using Apps Manager to Set Proxy Environment Variables
+
+The proxy environment variables set above using the Cloud Foundry CLI can also be set using the Apps Manager.
+
+### AppDynamics Nozzle app
+1. Navigate to the appdynamics space.
+1. Open the **Settings** tab for the `appdnozzle-<version>` app. 
+1. Click **REVEAL USER PROVIDED ENV VARS**.
+
+   ![](./images/reveal-user-envars.png)
+
+1. Add the proxy environment variables and click **SAVE**.
+
+   ![](./images/add-envars-save.png)
+
+1. Click the **Restart** button to restart the app.
+
+   ![](./images/restart-app-ui.png)
+
+### AppDynamics KPI Dashboard app 
+1. Navigate to the appdynamics space.
+1. Open the **Settings** tab for the `appdpcfdashboard-<version>` app. 
+1. Click **REVEAL USER PROVIDED ENV VARS**.
+
+    !!!! **Note:** Replace these Screenshots with KPI Dashboard App Images !!!!
+
+   ![](./images/reveal-user-envars.png)
+
+1. Add the proxy environment variables and click **SAVE**.
+
+   ![](./images/add-envars-save.png)
 
-    $ cf restage appdnozzle-<version>
-    $ cf restage  appdpcfdashboard-<version>
+1. Click the **Restart** button to restart the app.
 
-## Apps Manager Configuration
-Set the environment variables from the Apps Manager as well.
+   ![](./images/restart-app-ui.png)
```

Comment:
3 of 3, Need KPI Dashboard app image to replace Nozzle app

Reply:
@okilker I've added the 3 images to bottom of page: https://singularity.jira.com/wiki/spaces/~939476479/pages/589572732/Apps+Manager+instructions